thehive.json

```diff
@@ -39,6 +39,12 @@
             "data_type": "boolean",
             "default": false,
             "order": 1
+        },
+        "timezone": {
```

Comment:
Is it possible to make this a dropdown list?

Reply:
Hey @alexa-phantom, the data type is 'timezone', so it will be a dropdown only.